### ISSUE TITLE
Should support chaining

### ISSUE THREAD

**Issue by btford** [btford/ngmin#11]
```
angular.module('mod', []).
    controller('MyCtrl1', function ($scope) { ... }).
    controller('MyCtrl2', function ($scope) { ... });
```

**Comment by ajoslin** [comment]
+1! :)

**Comment by ghost** [comment]
Love your work Brian! The auto DI annotations are brilliant.
In addition to supporting existing chains it'd be cool to group methods by module and chain them e.g. `angular.module("foo").bar([...]),angular.module("foo").baz([...]),angular.module("foo").qux([...]);` could be minified to `angular.module("foo").bar([...]).baz([...]).qux([...]);`

**Comment by btford** [comment]
@zobier, I've been thinking about that as well, but I want to do some investigation to make sure that actually saves bytes. I'll create a new issue for this possible additional optimization.

And thanks! I'm glad you like it.

**Comment by btford** [comment]
Now available in `v0.3.0`.
